sr_robot_commander/src/sr_robot_commander/sr_arm_commander.py

```diff
@@ -53,7 +53,7 @@ def move_to_position_target(self, xyz, end_effector_link="", wait=True):
         @param end_effector_link - name of the end effector link
         @param wait - should method wait for movement end or not
         """
-        self.move_to_position_target(xyz, end_effector_link, wait=wait)
+        super(SrArmCommander, self).move_to_position_target(xyz, end_effector_link, wait=wait)
```

Comment:
Probably these methods were copied to RobotCommander at some point, but instead of deleting them from here they were modified to look like this.

I think we should delete them from here.